tests/integration/test_dnf.py

```diff
@@ -26,9 +26,9 @@ def test_query_dnf_and_package() -> None:
     assert package.name == "slurm-slurmd"
     assert package.arch == "x86_64"
     assert package.epoch is None
-    assert package.version == "22.05.6"
-    assert package.release == "3.el9"
-    assert package.full_version == "22.05.6-3.el9"
+    assert package.version == "22.05.9"
+    assert package.release == "1.el9"
+    assert package.full_version == "22.05.9-1.el9"
```

Comment:
Two things:

1) Why are these numbers changing?
2) It's good to have unit tests, but I think we should also have an integration test for this. We already have [`integration/test_snap.py`](https://github.com/canonical/operator-libs-linux/blob/main/tests/integration/test_snap.py), so shouldn't be hard to add to that.

Reply:
>     1. Why are these numbers changing?

Most probably the package got updated in the images we use. The assertions fail without those changes. I can revert that, but then the integration suite will fail.

>     2. It's good to have unit tests, but I think we should also have an integration test for this. We already have [`integration/test_snap.py`](https://github.com/canonical/operator-libs-linux/blob/main/tests/integration/test_snap.py), so shouldn't be hard to add to that.

Added.